cmd/ana/main_test.go

```
@@ -536,6 +536,108 @@ func TestRun_LeafUsageErrorReturned(t *testing.T) {
+			t.Fatalf("stderr: %q", buf.String())
+		}
+	})
+}
+
+// TestUpdateCmd_Help short-circuits on --help like every other leaf verb.
+func TestUpdateCmd_Help(t *testing.T) {
+	t.Parallel()
+	var out bytes.Buffer
+	stdio := cli.IO{Stdout: &out, Stderr: &bytes.Buffer{}}
+	err := (updateCmd{}).Run(context.Background(), []string{"--help"}, stdio)
+	if err != cli.ErrHelp {
```

golangci-lint (2.11.4)

[error] comparing with != will fail on wrapped errors. Use errors.Is to check for a specific error

(errorlint)